docs/engines/flink/flink-dml.md

```diff
@@ -189,4 +189,49 @@ Hint Options
-| other table parameters                           | (none)        | String   | No                                                                                                                               | All parameters of an Amoro table can be dynamically modified through SQL Hints, but they only take effect for this specific task. For the specific parameter list, please refer to the [Table Configuration](../configurations/). For permissions-related configurations on the catalog, they can also be configured in Hint using parameters such as [properties.auth.XXX in catalog DDL](../flink-ddl/#flink-sql)                                                                                                                                                                                                                                                                                                                                                                                                             |
\ No newline at end of file
+| other table parameters                           | (none)        | String   | No                                                                                                                               | All parameters of an Amoro table can be dynamically modified through SQL Hints, but they only take effect for this specific task. For the specific parameter list, please refer to the [Table Configuration](../configurations/). For permissions-related configurations on the catalog, they can also be configured in Hint using parameters such as [properties.auth.XXX in catalog DDL](../flink-ddl/#flink-sql)                                                                                                                                                                                                                                                                                                                                                                                                             |
+
+## Lookup join with SQL
+
+A lookup join is typically used to enrich a table with data that is queried from an external system. The join requires one table to have a processing time attribute and the other table to be backed by a lookup source connector.
+The lookup join uses the above Processing Time Temporal Join syntax with the right table to be backed by a lookup source connector.
+
+The following example shows the syntax to specify a lookup join.
+
+```sql
+-- Customers is backed by the JDBC connector and can be used for lookup joins
```

Comment:
Customers is backed by the JDBC connector.

Should we pick up a table using data lake table?